### PR TITLE
Add Arb/CoArb instances for Unit and Ordering

### PR DESCRIPTION
**Pull request by garyb**
/cc @paf31

### CODE REVIEW

File: src/Test/QuickCheck.purs [hunk 1]
```diff
@@ -100,7 +100,26 @@ instance arbString :: Arbitrary String where
...
--- | A newtype for `String` whose `Arbitrary` instance generated random 
+instance arbUnit :: Arbitrary Unit where
+  arbitrary = return unit
+
+instance coarbUnit :: CoArbitrary Unit where
+  coarbitrary _ = perturbGen 1
+
+instance arbOrdering :: Arbitrary Ordering where
+  arbitrary = do
+    n <- (3 *) <$> uniform
```
Comment (paf31) [inline src/Test/QuickCheck.purs]: We should probably provide a wrapper for `Enum` to do this more generically, but for now, I think we have a combinator to generate an integer in a range. I forget the name..

**Comment by paf31** [comment]
:+1: Looks good, I just have one comment above.

File: src/Test/QuickCheck.purs (hunk 1 above)
Comment (garyb) [inline src/Test/QuickCheck.purs]: I looked through the stuff in `Gen` but there wasn't anything obviously suitable. Could you have a look too?

Comment (paf31) [inline src/Test/QuickCheck.purs]: I think `chooseInt` does what you need.

Comment (garyb) [inline src/Test/QuickCheck.purs]: Duhh, right, yeah. Thanks ;)

**Pull request merged**